lib/sass/calcite-web/patterns/_accordion.scss

```diff
@@ -19,33 +30,46 @@
   }
 
   @mixin accordion-title() {
-    @include side-nav-title();
     cursor: pointer;
-    &:focus,
-    &:hover {
-      @include transition(all, $transition);
-      background-color: $blk-010;
-      outline: none;
```

Comment:
The structure of this makes the clickable area of the accordion-title smaller than the implied area. We should add the padding to both the title, and the content, rather than on the section itself.

![Screen Shot 2019-03-26 at 9 49 49 AM](https://user-images.githubusercontent.com/4733155/55016897-e535fa80-4fac-11e9-90f5-f4e927c8ace4.png)

Reply:
This one should be good